rclrs/src/node/builder.rs

```diff
@@ -40,6 +44,10 @@ pub struct NodeBuilder {
     context: Arc<Mutex<rcl_context_t>>,
     name: String,
     namespace: String,
+    domain_id: usize,
```

Comment:
The domain ID isn't part of the node_options anymore from galactic onwards. I think it now is an attribute of the context, not the node: https://github.com/ros2/rcl/pull/708

So, probably better to remove this. Sorry, I think I explicitly suggested this on the issue!

Reply:
Or, instead of removing, you can gate it with `#[cfg(ros_distro = "foxy")]`.

Reply:
Thank you for letting me know about `domain_id`.
Please tell me where the description or documentation about `domain_id` is in context now.

Reply:
In newer distros it's part of the `init_options` that are passed to `rcl_init()` (called in the `Context::new` method).

Reply:
Probably it's best to leave the `domain_id` for a separate PR.